core/tests/org.fusesource.ide.camel.tests.util/src/org/fusesource/ide/camel/tests/util/CommonTestUtils.java

```diff
@@ -29,14 +30,18 @@
  */
 public class CommonTestUtils {
 	
+	private CommonTestUtils() {
+		//static access only
+	}
+	
 	/**
 	 * closes all editors, perspectives, the welcome page and then switches to fuse integration perspective
 	 * and creating a screenshot folder
 	 * 
 	 * @param screenshotFolder	the folder to store screenshots
 	 * @throws Exception
```

Comment:
Please update the exception to WorkbenchException. Moreover, could you specify when this exception is thrown, please?